zerver/lib/push_notifications.py

```diff
@@ -1050,6 +1063,13 @@ def handle_remove_push_notification(user_profile_id: int, message_ids: List[int]
             send_android_push_notification(user_identity, android_devices, gcm_payload, gcm_options)
         if apple_devices:
             send_apple_push_notification(user_identity, apple_devices, apns_payload)
+        do_increment_logging_stat(
+            user_profile.realm,
+            COUNT_STATS["mobile_pushes_sent::day"],
+            None,
+            timezone_now(),
+            increment=len(android_devices) + len(apple_devices),
+        )
```

Comment:
The math here and in the other code path are wrong in case any devices are invalid and were deleted; I think `send_apple_push_notification` and `send_android_push_notification` return the correct numbers to use.

Reply:
Good catch, tweaked this and the test logic to test that we actually use the returned success values for the increment